### PR TITLE
Small function fixes + SuitPage graphic fix.

### PR DESCRIPTION
**Pull request by tylerblurie**

### CODE REVIEW

File: toontown/spellbook/MagicWordIndex.py [hunk 1]
```diff
@@ -1890,17 +1890,23 @@ def handleWord(self, invoker, avId, toon, *args):
...
+        if not 0 <= amt <= 10500:
+            return "Can't set {0}'s experience level to {1}! Specify a value between 0 and 10,500.".format(toon.getName(), amt)
 
         if track in maxed:
             for track in tracks:
                 toon.experience.setExp(track, 10000)
             toon.b_setExperience(toon.experience.makeNetString())
             return "Maxed all of {}'s gag tracks.".format(toon.getName())
+        elif track == 'all':
+            for individualTrack in xrange(len(tracks)):
```
Comment (DarthMDev) [inline toontown/spellbook/MagicWordIndex.py]: range is better as xrange is removed in python3

Comment (2Epik4u) [reply toontown/spellbook/MagicWordIndex.py]: this source isn't in python3 though

Comment (tylerblurie) [reply toontown/spellbook/MagicWordIndex.py]: @DarthNihilus1 Good point. I've gone ahead and amended a commit to fix that.

**Review by DarthMDev — APPROVED** [review]

**Comment by GivingCash** [comment]
I have been away for some time now, any help on getting started again?

**Pull request merged**